benchmarks/filters/voxel_grid.cpp

```diff
@@ -44,28 +45,73 @@ BM_ApproxVoxelGrid(benchmark::State& state, const std::string& file)
   }
 }
 
+static void
+BM_Experimental_VoxelGrid(benchmark::State& state, const std::string& file)
```

Comment:
https://github.com/google/benchmark#templated-benchmarks

The code repetition can be improved

Reply:
I didn't look at your suggestion before I start implementing 😅
But seems that `BENCHMARK_TEMPLATE` couldn't achieve our goal https://stackoverflow.com/questions/38023017/google-benchmark-benchmark-template-syntax